wpilibc/src/main/native/cpp/AnalogTrigger.cpp

```diff
@@ -109,8 +109,7 @@ bool AnalogTrigger::GetTriggerState() {
 std::shared_ptr<AnalogTriggerOutput> AnalogTrigger::CreateOutput(
     AnalogTriggerType type) const {
   return std::shared_ptr<AnalogTriggerOutput>(
-      new AnalogTriggerOutput(*this, type),
-      wpi::NullDeleter<AnalogTriggerOutput>());
+      new AnalogTriggerOutput(*this, type));
```

Comment:
Use std::make_shared instead.

Reply:
std::make_shared can't access the protected AnalogTriggerOutput ctor.

Reply:
Ok.  The solution we use in other places for that is a private (or protected in this case) `struct private_init {};` and pass that as a parameter to the (public) constructor.  I'm not sure it's worth it to do it here though.